language/move-lang/tests/move_check/parser/spec_parsing_ok.move

```diff
@@ -0,0 +1,97 @@
+// These are some basic parsing tests for specifications which are expected to succeed.
+// Full testing of spec parsing correctness is done outside of this crate.
+//
+// Note that even though we only test parsing, we still need to ensure that the move code (not the specification)
+// is type checking correctly, because with no parsing error, the test harness
+// will run subsequent phases of the move-lang compiler.
+//
+// For parse failures, see the `spec_*_fail.move` test cases.
+
+module M {
+    synthetic expected_market_cap: u64;
+    synthetic other: bool;
+
+    resource struct T{x: u64}
+    resource struct R{x: u64}
+
+    struct S {
+        x : u64,
+        invariant x > 0,
+        y: u64,
+        invariant update old(y) < x,
+        invariant pack sum = sum + x,
```

Comment:
`sum` -> `expected_market_cap`?

Reply:
Good catch, I made this test example more meaningful.